dask_ml/model_selection/_incremental.py

```diff
@@ -203,8 +203,11 @@ def get_futures(partial_fit_calls):
     _scores = {}
     _specs = {}
 
-    d_partial_fit = dask.delayed(_partial_fit)
-    d_score = dask.delayed(_score)
+    def d_partial_fit(*args, **kwargs):
+        return client.submit(_partial_fit, *args, **kwargs)
+
+    def d_score(*args, **kwargs):
+        return client.submit(_score, *args, **kwargs)
```

Comment:
I would prefer using `client.submit` directly and avoiding these function definitions if that's ok.  

Motivation here: http://matthewrocklin.com/blog/work/2019/06/23/avoid-indirection